specs/server.test.js

```diff
@@ -13,17 +14,45 @@ describe('Server', () => {
     done();
   });
 
-  test('GET request should not contain listing id', (done) => {
+  test('GET request body should not contain listing id', (done) => {
     request({
       url: `${url}/listings/1`,
       method: 'GET',
     }, (error, response, body) => {
-      const bodyjson = JSON.parse(body);
-      for (let i = 0; i < bodyjson.length; i += 1) {
-        expect(bodyjson[i].id).not.toBe(1);
+      const results = JSON.parse(body);
+      for (let i = 0; i < results.length; i += 1) {
+        expect(results[i].id).not.toBe(1);
```

Comment:
What is this testing for?

Reply:
The similar listings items shouldn't contain the listing the page is on